src/main/java/jmgomez/apipolicy/model/Policy.java

```diff
@@ -15,9 +17,9 @@ public class Policy {
     @JsonProperty("descripcion")
     private String description;
     @JsonProperty("coberturas")
-    private String[] coverages;
+    private List<String> coverages;
     @JsonProperty("condiciones")
-    private String[] conditions;
+    private List<String> conditions;
     @JsonProperty("siniestros")
-    private Accident[] accidents;
+    private List<Accident> accidents;
```

Comment:
Why are you setting this? Where is it being used? I know for a fact the final services aren't asnwering with the accidents related to a policy in the policy endpoint... also there's no setAccidents in code (not that I wanted it)... so... why do we have this in our model?